### ISSUE TITLE
load_balancer errors on :destroy action

### ISSUE THREAD

**Issue by vinyar** [chef/chef-provisioning#278]
When load balancer exists, calling it with action :destroy throws an error stating that :destroy is not a valid action. LB is in fact deleted but chef run is halted.

Changing it to action :delete throws a correct compilation error stating that action :destroy should be used.

ChefDK version 0.4.0
chef-provisioning (0.18, 0.17)
chef-provisioning-aws (0.2.1)
chef-provisioning-azure (0.1)
chef-provisioning-fog (0.12)
chef-provisioning-vagrant (0.8.1)

```ruby
load_balancer "alexv-webserver-lb" do
  action :destroy
end
```
Error:
```ruby
  - Deleting EC2 ELB http://localhost:8889/data/loadbalancers/alexv-webserver-lb
    ================================================================================
    Error executing action `destroy` on resource 'load_balancer[alexv-webserver-lb]'
    ================================================================================

    Chef::Exceptions::ValidationFailed
    ----------------------------------
    Option action must be equal to one of: nothing, create, delete, nothing!  You passed :destroy.

    Resource Declaration:
    ---------------------
    # In /Users/alexvinyar/Documents/Projects/chef-provisioning/.chef/local-mode-cache/cache/cookbooks/webserver/recipes/deprovision.rb

     41: load_balancer "#{name}-webserver-lb" do
     42:   action :destroy
     43: end
     44:

    Compiled Resource:
    ------------------
    # Declared in /Users/alexvinyar/Documents/Projects/chef-provisioning/.chef/local-mode-cache/cache/cookbooks/webserver/recipes/deprovision.rb:41:in `from_file'

    load_balancer("alexv-webserver-lb") do
      action [:destroy]
      retries 0
      retry_delay 2
      default_guard_interpreter :default
      chef_server {:chef_server_url=>"http://localhost:8889", :options=>{:client_name=>"alexv", :signing_key_filename=>"/Users/alexvinyar/Documents/Projects/chef-provisioning/.chef/keys/chef_default"}}
      driver "aws"
      declared_type :load_balancer
      cookbook_name "webserver"
      recipe_name "deprovision"
    end
```
error 2
```ruby
Chef::Exceptions::ValidationFailed
----------------------------------
Option action must be equal to one of: nothing, create, destroy!  You passed :delete.
```

**Comment by jkeiser** [comment]
I am certainly perplexed.  Setting 1.0 for next week's purge.  @vinyar could you give stack by any chance?  It should have a line where it shows the file the stacktrace was dumped to.  Thanks :)

**Comment by vinyar** [comment]
@jkeiser no stacktrace is thrown. Running chef client in debug mode doesnt produce any useful output either.

**Comment by jkeiser** [comment]
Fixed with chef/chef-provisioning#273.  Please feel free to reopen if it's still happening, but everything in this bug is consistent with that fix :)  That fix is released to the wild in the latest chef-provisioning driver, so update your gems and give it a whirl!
